### PR TITLE
Various: Property updates

### LINKED ISSUE #1222: Duplicate: two Solihull localities (both marked mz:is_current = -1)
https://spelunker.whosonfirst.org/id/1126055031/ and https://spelunker.whosonfirst.org/id/1125891303/

Both are Solihull (a city in the UK) - both have `mz:is_current = -1`

2018-06-14 at 11 43 am

Related: #1221

### PR DESCRIPTION
**Pull request by stepps00**
Fixes #1222, #1221, #1206, #1236, #1248, and #1249.

This PR updates records in recent "low hanging" issues. Each issue fix is taken care of in a single commit.

- 131c9f67010a2dd5fbb25c82949b120615fe3611: updates Poland descendants with the correct ISO and WOF country codes.
- 405bbdb2f23ab8b5e860559a7bf20460f08d1cf0: fixes two duplicate locality points, adds properties/concordances, and supersedes work.
- a565aec31abec2b93f69f1c94ce8a76570008f2c: updates hierarchy of a stranded neighbourhood record.
- 3eeb09fd4fb171a3303ca74e7d2a55bc19f141a6: PIPs the locality point of Hong Kong to add missing parents to the hierarchy.
- d7f39ad91440a2a589205d90e947a8533edc4818: Corrects properties, name translations, and concordance values in the Multnomah neighbourhood.
- 20f6660e2f1411555da8ca0ba6e23406d07de0ab: Updates ISO and WOF country codes for record on the US/MX border.

All PIP work included, should be a straightforward merge once approved.

### CODE REVIEW

File: data/112/589/130/3/1125891303.geojson [hunk 1]
```diff
@@ -107,15 +114,17 @@
...
     "wof:name":"Solihull",
     "wof:parent_id":404446879,
     "wof:placetype":"locality",
     "wof:population":96267,
     "wof:population_rank":8,
     "wof:repo":"whosonfirst-data",
     "wof:superseded_by":[],
-    "wof:supersedes":[],
+    "wof:supersedes":[
+        1126055031
```
Comment (nvkelso) [inline data/112/589/130/3/1125891303.geojson]: This feature was WOE id of 35110 so please add that as a woe concordance to this record it's being superseded into.

File: data/859/080/29/85908029.geojson [hunk 2]
```diff
@@ -83,15 +81,18 @@
     "wof:geomhash":"a9b011c77d24b3e223b5433b8ec6c8e2",
     "wof:hierarchy":[
         {
+            "continent_id":-1,
+            "country_id":-1,
```
Comment (nvkelso) [inline data/859/080/29/85908029.geojson]: Shouldn't this have a hierarchy including HK which is a `country`? https://spelunker.whosonfirst.org/id/85632483/

Comment (stepps00) [reply data/859/080/29/85908029.geojson]: This record's lat/long falls outside of a locality, so the `-1` values are populated for all its parents.. I'll add revgeo coords and re-PIP.

File: data/112/589/130/3/1125891303.geojson (hunk 1 above)
Comment (stepps00) [reply data/112/589/130/3/1125891303.geojson]: :+1:

**Comment by stepps00** [comment]
Requested changes made through the two most recent commit.

Merging and Closing.

**Pull request merged**